sdk/ml/azure-ai-ml/tests/conftest.py

```diff
@@ -59,7 +59,7 @@ def fake_datastore_key() -> str:
 
 @pytest.fixture(autouse=True)
 def add_sanitizers(test_proxy, fake_datastore_key):
-    add_remove_header_sanitizer(headers="x-azureml-token,Log-URL")
+    add_remove_header_sanitizer(headers="x-azureml-token,Log-URL,Authorization")
```

Comment:
Is this for recordings? (I thought only CLI had recordings?) 

Are there any files that need to be re-recorded or scrubbed?

Reply:
Yes, this is for recordings. After the migration we have SDK recordings for e2e tests too.
This change was needed because one of the batch deployment tests could possibly expose an authorization key.